The overflow plugin needed the hunk below but it broke out-of-tree builds
Since we don't use the overflow plugin we revert it.

diff --git a/Makefile b/Makefile
index 073f74f..b379941 100644
--- a/Makefile
+++ b/Makefile
@@ -357,8 +358,8 @@ CFLAGS_GCOV	= -fprofile-arcs -ftest-coverage
 
 # Use LINUXINCLUDE when you must reference the include/ directory.
 # Needed to be compatible with the O= option
-LINUXINCLUDE    := -isystem arch/$(hdr-arch)/include \
-                   -isystem arch/$(hdr-arch)/include/generated -isystem include \
+LINUXINCLUDE    := -I$(srctree)/arch/$(hdr-arch)/include \
+                   -Iarch/$(hdr-arch)/include/generated -Iinclude \
                    $(if $(KBUILD_SRC), -I$(srctree)/include) \
                    -include $(srctree)/include/linux/kconfig.h
 
